gaphor/diagram/propertypages.py

```diff
@@ -312,6 +311,8 @@ def __init__(self, subject: NamedElement):
         self.watcher = subject.watcher() if subject else DummyEventWatcher()
 
     def construct(self):
+        from gaphor import UML
```

Comment:
Do we need a comment here explaining why this import can't be at the top of the module?

Reply:
Maybe it needs to move to the UML package as well. I'm not sure yet.

Reply:
I moved the NamedElement editor to the UML package.